pac4j-http/src/main/java/org/pac4j/http/credentials/CredentialUtil.java

```diff
@@ -0,0 +1,43 @@
+package org.pac4j.http.credentials;
+
+import static java.lang.String.copyValueOf;
+
+import org.apache.commons.codec.binary.Hex;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class contains utility methods related to credential encryption algorithms
+ * (MD5 for http digest)
+ *
+ * @author Mircea Carasel
+ */
+public class CredentialUtil {
+
+    /**
+     * Defined in rfc 2617 as H(data) = MD5(data);
+     *
+     * @param data data
+     * @return MD5(data)
+     */
+    public static String H(String data) {
```

Comment:
Is there a reason this method is called H? I understand that's the spec's name, but surely, we could pick a name that is more sensible and leave the javadocs to explain the spec ref?